src/main/java/org/kcctl/command/GetOffsetsCommand.java

```diff
@@ -0,0 +1,88 @@
+/*
+ *  Copyright 2021 The original authors
```

Comment:
Logged https://github.com/kcctl/kcctl/issues/340 for getting rid of the year there and use a simpler heade altogether.